src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js

```diff
@@ -0,0 +1,36 @@
+const HttpErrorService = ($state, $translate, notification) => ({
+    handleError: function(event, toState, toParams, fromState, fromParams, error) {
+        switch (error.status) {
+            case 404:
+               this.handle404Error(event, error);
+               break;
+            case 403:
+               this.handle403Error(error);
```

Comment:
Please, be consistent and give the same API for all the handlers: `(event, error)` or `(error, event)`.
I'm even in favor of `(error, event, toState, toParams, fromState, fromParams)`.

It will be more customable for the next developers and the doc will be more clear too.